firefox-ios/Client/Frontend/Home/TopSites/DataManagement/UnifiedAds/UnifiedAdsNetwork.swift

```diff
@@ -0,0 +1,30 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Foundation
+
+/// Used only for sponsored tiles content and telemetry. This is aiming to be a temporary API
+/// as we'll migrate to using A-S for this at some point next year
```

Comment:
Nit: `next year` requires us to look up when the code was written. Shall we replace this with `in 2025`?